src/ringcentral-web-phone.js

```diff
@@ -836,74 +864,7 @@
-                throw new SIP.Exceptions.InvalidStateError(session.status);
-            }
-
-            // normalizeTarget allows instances of SIP.URI to pass through unaltered,
-            // so try to make one ahead of time
-            try {
-                target = SIP.Grammar.parse(target, 'Refer_To').uri || target;
-            } catch (e) {
-                session.logger.debug(".refer() cannot parse Refer_To from", target);
-                session.logger.debug("...falling through to normalizeTarget()");
-            }
-
-            // Check target validity
-            target = session.ua.normalizeTarget(target);
-            if (!target) {
-                throw new TypeError('Invalid target: ' + originalTarget);
-            }
-
-            extraHeaders = extraHeaders.concat(session.ua.defaultHeaders).concat([
-                'Contact: ' + session.contact,
-                'Allow: ' + SIP.UA.C.ALLOWED_METHODS.toString(),
-                'Refer-To: ' + target
-            ]);
-
-            // Send the request
-            session.sendRequest(SIP.C.REFER, {
-                extraHeaders: extraHeaders,
-                body: options.body,
-                receiveResponse: function(response) {
-                    var timeout = null;
-                    if (response.status_code === 202) {
-                        var callId = response.call_id;
-
-                        var onNotify = function(request) {
-                            if (request.call_id === callId) {
-                                var body = request && request.body || '';
-                                switch (true) {
-                                    case /1[0-9]{2}/.test(body):
-                                        request.reply(200);
-                                        break;
-                                    case /2[0-9]{2}/.test(body):
-                                        session.terminate();
-                                        clearTimeout(timeout);
-                                        session.removeListener('RC_SIP_NOTIFY', onNotify);
-                                        resolve();
-                                        break;
-                                    default:
-                                        reject(body);
-                                        break;
-                                }
-                            }
-                        };
-
-                        timeout = setTimeout(function() {
-                            reject(new Error('Timeout: no reply'));
-                            session.removeListener('RC_SIP_NOTIFY', onNotify);
-                        }, responseTimeout);
-                        session.on('RC_SIP_NOTIFY', onNotify);
-                    }
-                    else {
-                        reject(new Error('The response status code is: ' + response.status_code + ' (waiting for 202)'));
-                    }
-                }
-            });
-
```

Comment:
RC_SIP_NOTIFY can now be removed completely.

Reply:
Removed